src/dtree.ml

```diff
@@ -0,0 +1,192 @@
+open Terms
+open Extra
+
+(** Type of the leaves of the tree.  See {!file:terms.ml}, {!recfield:rhs}. *)
+type action = (term_env, term) Bindlib.mbinder
+
+(** Tree type.  Leaves are the targets of the rewriting, i.e. when a leaf is
+    reached, the matched pattern is rewrote to the the leaf.  The
+    {!const:Node}[(i, t)] constructor allows to perform a switch among the
```

Comment:
Node takes a node_data record as argument, not a pair.

Reply:
Doc rewritten in 1ade3d6